spec/ewb/IEC61970/Base/Wires/Conductor.yaml

```diff
@@ -7,10 +7,10 @@ attributes:
   description: Segment length for calculating line section capabilities.
 - name: designTemperature
   type: Temperature
-  description: [ZBEX] The temperature in degrees Celsius for the network design of this conductor. Values are limited to integers.
+  description: \[ZBEX\] The temperature in degrees Celsius for the network design of this conductor. Values are limited to integers.
```

Comment:
Have to escape that brackets :(, hence why I want to make it a real attribute so we can make the markdown look good.